File: src/main/java/cn/edu/zqu/hr_system/project/model/entities/Agreement.java
package cn.edu.zqu.hr_system.project.model.entities;

import com.baomidou.mybatisplus.annotation.FieldFill;
import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.Data;

import java.sql.Date;

@Data

public class Agreement {
  @TableId(type = IdType.AUTO)
  Long id; //
  Long uid; // 用户编号


  int wage; // 基本工资
  //  int endowment; // 养老保险
//  int medical; // 医疗保险
//  int unemployment; // 失业保险
//  int employment; // 工伤保险
//  int maternity; // 生育保险
  int insurance;
  int housingFund; // 住房公积金

  Date beginTime; // 开始时间
  Date endTime; // 结束时间

  char status; // 状态

  @TableField(fill = FieldFill.INSERT)
  long createBy; // 创建者
  @TableField(fill = FieldFill.INSERT)
  Date createTime; // 创建时间

  @TableField(fill = FieldFill.INSERT_UPDATE)
  long updateBy; // 更改者
  @TableField(fill = FieldFill.INSERT_UPDATE)
  Date updateTime; // 更改时间


}

File: browser/src/model/agreement.ts
export interface AgreementItem{
  id:number; //
  uid:number; // 用户编号
  wage:number; // 基本工资
  // endowment:number; // 养老保险
  // medical:number; // 医疗保险
  // unemployment:number; // 失业保险
  // employment:number; // 工伤保险
  // maternity:number; // 生育保险
  insurance:number;
  housingFund:number; // 住房公积金
  beginTime:Date; // 开始时间
  endTime:Date; // 结束时间
  status:string; // 状态
}
